src/views/Alteration.vue

```diff
@@ -15,7 +15,7 @@
 
       <current-directors class="mt-10" />
 
-      <share-structure class="mt-10" />
+      <share-structures class="mt-10" />
```

Comment:
Why is it plural?

Reply:
🤷  Because there is multiple share structures? hehe i guess that is open to interpretation though! 
Also, didn't want to conflict with the shared component.

Reply:
Isn't there only 1 share structure?

We can worry about this later. There's enough other stuff to do !!!